nixos/modules/virtualisation/amazon-config.nix

```diff
@@ -1,5 +1,3 @@
-{ config, pkgs, modulesPath, ... }:
-
 {
-  imports = [ "${modulesPath}/virtualisation/amazon-image.nix" ];
+  imports = [ ./amazon-image.nix ];
```

Comment:
To be honest, I do not really follow what is the reason of having this file, shouldn't all of these be part of nixops?